### ISSUE TITLE
seems missing mbedtls android lib?

### ISSUE THREAD

**Issue by muyufan** [vlang/vab#266]
hello, I try to build a v example file which using http request into apk, but on android, logcat show below error message.

"04-11 18:09:37.357 26605 26605 E AndroidRuntime: java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "mbedtls_entropy_func" referenced by "/data/app/io.v.android-ihK8P-u8HB98ZcUZ9wgnCw==/lib/arm64/libv_test_app.so"...

does it mean that start that apk, it try to loading mbedtls library, but it couldn't find it?

how should I include maybe the "static mbedtls library" in to apk to avid this error? or any other suggestion? thank you.

**Comment by Larpon** [comment]
Yes that's odd. SSL on Android has a messy history unfortunately. But ofc SSL should work somehow

**Comment by larpon** [comment]
@muyufan I have recently improved the way `vab` compiles C dependencies - can you please try again with latest `vab` and report back if the problem still exist?
